api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistUserRepository.kt

```diff
@@ -15,10 +16,10 @@ import javax.inject.Inject
 class AnilistUserRepository @Inject constructor(
     @AuthorizedClient private val apolloClient: ApolloClient
 ) {
-    fun fetchViewer(): Flow<Result<ViewerQuery.Viewer>> {
+    fun fetchViewer(useNetwork: Boolean): Flow<Result<ViewerQuery.Viewer>> {
         return apolloClient
             .query(ViewerQuery())
-            .fetchPolicy(FetchPolicy.CacheAndNetwork)
+            .fetchPolicy(if (useNetwork) FetchPolicy.NetworkFirst else FetchPolicy.CacheFirst)
             .toFlow()
```

Comment:
If our cache isn't reactive, and our network isn't reactive, what's the point of returning a Flow?

Reply:
I'm not entirely sure if they are/aren't tbh :/

Reply:
We started using flows in #67, I'll see if this provides any benefits and revert it if not in a future PR.